lib/ansible/modules/packaging/os/apt_key.py

```diff
@@ -32,7 +32,7 @@
         required: false
         default: none
         description:
-            - identifier of key. Including this allows check mode to correctly report the changed state.
+            - identifier of key. Including this allows check mode to correctly report the changed state. Required when C(state) is set to C(absent).
```

Comment:
Descriptions need to start with a Uppercase character. And should be a full English sentence (or description).
So something like:

    The identifier of the key. ...

I would add the remark about when it is required on a separate bulle, at the end.

    - This parameter is required when C(state) is set to C(absent).